pytest_splinter/plugin.py

```diff
@@ -321,14 +325,19 @@ def make_screenshot_on_failure():
                 else:
                     screenshot_dir = tmpdir.mkdir('screenshots').strpath
                 screenshot_path = os.path.join(screenshot_dir, screenshot_file_name)
-                browser.driver.save_screenshot(screenshot_path)
-                with open(screenshot_path) as fd:
-                    if slaveoutput is not None:
-                        slaveoutput.setdefault('screenshots', []).append({
-                            'class_name': classname,
-                            'file_name': screenshot_file_name,
-                            'content': fd.read()
-                        })
+                try:
+                    LOGGER.info('Saving screenshot to %s' % (screenshot_path,))
```

Comment:
It won't / should not affect production systems / logs, but is used only during tests - and in that case it's a good/useful information.